Xamarin.Forms.Controls/GalleryPages/PlatformTestsGallery/PlatformTestsConsole.xaml.cs

```diff
@@ -28,6 +29,7 @@ public partial class PlatformTestsConsole : ContentPage
 		int _testsRunCount = 0;
 
 		readonly PlatformTestRunner _runner = new PlatformTestRunner();
+		DisplaySettings _displaySettings;
```

Comment:
These changes make it so if there is a test failure all the passed tests become non visible by default. This lets me see on the app center screen shot what is failing